governator-annotations/src/main/java/com/netflix/governator/annotations/AutoBind.java

```diff
@@ -26,11 +26,14 @@
  * Annotation binding that combines with Governator's classpath scanning and a bound
  * AutoBindProvider to automatically/programmatically bind fields and constructor/method
  * arguments
+ * 
+ * @deprecated Deprecated in favor of standard Guice modules
```

Comment:
I'd prefer we link to a page that lays out just the issue and fix for the user rather than having to parse a full thread to figure out what's relevant.